File: src/Practica1Entornos.java
import java.util.Scanner;

public class Practica1Entornos {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Programa mensaje en Java");

        System.out.print("¿Cómo se llama? ");
        String nombre = input.nextLine();

        System.out.print("¿En qué año nació? ");
        int nacimiento = input.nextInt();

        int edad = 2025 - nacimiento;
        String lenguaje = "Java";

        System.out.println("Hola " + nombre + ", tienes " + edad + " años. Este programa está hecho en el lenguaje de programación: " + lenguaje);

        input.close();
    }
}

File: src/Practica1Entornos.c
#include <stdio.h>

int main() {
    char nombre[50];
    int nacimiento, edad;

    printf("Programa mensaje en C\n");

    printf("¿Cómo se llama?\n ");
    scanf("%49s", nombre);

    printf("¿En qué año nació?\n ");
    scanf("%d", &nacimiento);

    edad = 2025 - nacimiento;
    printf("Hola %s, tienes %d años. Este programa está hecho en el lenguaje de programación: C]", nombre, edad);

    return 0;
}
